app/services/katello/candlepin/upstream_consumer.rb

```diff
@@ -0,0 +1,28 @@
+module Katello
+  module Candlepin
+    class UpstreamConsumer
+      def initialize(organization)
+        @organization = organization
+      end
+
+      def simple_content_access_eligible?
```

Comment:
Based on the output of the new candlepin API (see the PR in description), confirm that this will parse and handle the response appropriately